pkg/ccl/backupccl/key_rewriter.go

```diff
@@ -258,23 +258,33 @@ func (kr *KeyRewriter) RewriteKey(key []byte) ([]byte, bool, error) {
 	return rekeyed, ok, err
 }
 
-// rewriteTableKey rewrites the table IDs in the key.
-// It assumes that any tenant ID has been stripped from the key so it operates
-// with the system codec. It is the responsibility of the caller to either
-// remap, or re-prepend any required tenant prefix.
-func (kr *KeyRewriter) rewriteTableKey(key []byte) ([]byte, bool, error) {
+// ErrImportingKeyError indicates the current key is apart of an in-progress import
+var ErrImportingKeyError = errors.New("Importing Key")
+
+// checkAndRewriteTableKey rewrites the table IDs in the key. It assumes that
+// any tenant ID has been stripped from the key so it operates with the system
+// codec. It is the responsibility of the caller to either remap, or re-prepend
+// any required tenant prefix.
```

Comment:
Can we add a few sentences about the new parameter we have added, and what its used for.

Reply:
Also maybe mention that a zero value skips the checks associated with the wallTime.